scripts/linters/css_linter.py

```diff
@@ -58,6 +59,34 @@ def all_filepaths(self):
         """Return all filepaths."""
         return self.files_to_lint
 
+    @staticmethod
+    def _get_trimmed_error_messages(css_lint_output):
+        """Remove extra bits from stylelint error messages.
+
+        Args:
+            css_lint_output: str. Output returned by the css linter.
+
+        Returns:
+            str. A string with the trimmed error messages.
+        """
+        error_messages = []
```

Comment:
Let's call this trimmed_error_messages. Ditto for other functions.

(In general, what you are processing is error messages, so being clear about the distinction seems useful.)

Reply:
Done.